### DISCUSSION TITLE
[FR] Support Current Raspbian install

### DISCUSSION THREAD

**Discussion by bannsec** [pivpn/pivpn#1414] [Feature requests]
**Is your feature request related to a problem? Please describe.**
No?

**Describe the solution you'd like**
I'd like to have support to install pivpn on Raspbian.

**Describe alternatives you've considered**

**Additional context**
I just installed a pi today with the current official raspian installer and image. My lsb release is:

```
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

Turns out, you're only checking for Ubuntu names. Bullseye is a Debian release from October 2021.

https://www.debian.org/releases/bullseye/

**Comment by coolapso** [comment]
This needs to be done, true. 

Pull Requests are Welcome tho!! 
I'll gladly review and test them. 

However your statement is wrong, we are not only checking for Ubuntu names, otherwise this project couldn't even be focused on raspberrypi. 

Here you can see we currently support both Stretch and Buster

https://github.com/pivpn/pivpn/blob/master/auto_install/install.sh#L266

https://github.com/pivpn/pivpn/blob/master/auto_install/install.sh#L273

**Comment by Silent-Hunter** [comment]
I've been using pivpn on bullseye, and it seems to work just fine. I've only used the Wireguard setting though.

**Comment by coolapso** [comment]
support added and pushed to **test** branch 

https://github.com/pivpn/pivpn/commit/e42c339baa94adf0edddadeda1a43ee818b9ea9b

**Comment by coolapso** [comment]
Released to master branch. 
https://github.com/pivpn/pivpn/blob/master/CHANGELOG.md#december-03rd-2021
